WriterDesk1/src/pages/Settings.js

```diff
@@ -265,31 +267,31 @@ const Settings = () => {
                     <Typography sx={{alignSelf: 'center', alignContent:'inline'}}>
                         View the <a className='userDataLinkPopup' onClick={() => {setShowUserDataPopup(true)}} >user data agreement</a>.
                     </Typography>
-                    {showUserDataPopup && <AlertDialog title = "User data agreement" 
+                    {showUserDataPopup && <AlertDialog title = "User data agreement"
                         text = "The data conditions allow the application to record user data. The user data includes the URL of the page, the location on the screen, and the timestamp of each click from the user.  The user data is only used to improve the automatic feedback generated within the application. If the purpose of the data changes, the application will ask the user again for permission to save their user data. This data does not include necessary sign-up information, such as the university email address and the password, since that is saved to ensure the functionalities of the application. The sign-up information is not used for any other purposes than logging into the application. The application is still fully available when refusing the data conditions. If the user refuses permission, no user data will be recorded of this user until the moment that they accept the data settings in the future. If the user accepts the permission at first but later revokes the permission, their recorded user data is deleted and no user data will be recorded of this user, until the moment that they accept the data settings in the future. Users can retrieve the recorded user data so far at any time. Users can ask questions regarding their data by sending a mail to i.l.h.rutten@student.tue.nl; a response will be provided within a month."
                         buttonAgree={<Button onClick={() => {setShowUserDataPopup(false)}}>I understand</Button>}
                     />}
                 </div>
                 <br />
                 <BlueButton idStr='downloadMyUserData' onClick={() => {handleOwnUserData()}}>Download my user data</BlueButton>
                 <br />
+                <br />
                 <Typography variant='h5' style={{color: '#44749D'}}>
                     Change password
                 </Typography>
                 <br />
-                <TextField value = {oldPassword} onChange={(e) => {setOldPassword(e.target.value); 
-                setFormError(""); setSuccessMessage("")}} id='currPass' label='Insert current password.' 
-                variant='outlined' type = 'password' style={{marginBottom: '1vw'}} />
+                <TextField value = {oldPassword} onChange={(e) => {setOldPassword(e.target.value);
+                setFormError(""); setSuccessMessage("")}} id='currPass' label='Insert current password.'
+                variant='outlined' type = 'password' style={{width: '260px'}} helperText={' '}/>
```

Comment:
Is deze helper text nodig?

Reply:
Ja, is nodig om dezelfde afstand tussen de textfields te krijgen